### ISSUE TITLE
Submitting assignment

### ISSUE THREAD

**Issue by cwaiz621** [cwaiz621/simpletodo_crystalwai#1]
Outstanding issue: 
Since the task item is just a string and does not have a boolean for completion state, when the view refreshes, I lose all the checkmarks. To make that feature work, I would have to change the Task object into a task object, instead of just being a string. 

@codepathreview @codepath

**Comment by nesquena** [comment]
Hi Crystal,

Thanks for applying! Based on your LinkedIn it doesn't seem that you meet the [selection criteria for the course](http://courses.codepath.com/snippets/intro_to_android/selection_criteria). Can you review the criteria and confirm or clarify. I'd encourage you to review [the resources here](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me) for a good starting point in that case as much of our content is openly available.

**Comment by cwaiz621** [comment]
Hello Nathan,

I am not currently a professional developer, but would definitely love to
move in that direction career wise. I have coding experience from classes
in college. If it is not possible to attend the class in person, would it
be possible to attend as a remote observer?

Thanks,
Crystal

**Comment by nesquena** [comment]
OK, I will be in touch about adding you as an observer in the next few weeks. Happy to have you participate as a remote self-paced observer.

**Comment by cwaiz621** [comment]
Hey Nathan,

I wanted to follow up and see if you could add me as a observer for the
android boot camp course.

Thanks,
Crystal

On Tue, Jul 28, 2015 at 12:32 AM, Nathan Esquenazi <notifications@github.com

**Comment by nesquena** [comment]
Yep you are all set, observer group will kick off in just 2 weeks, more details to follow very soon. Sorry for the delay in communication.
